app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -257,6 +269,16 @@ export default function Navigation() {
               label={FORUM}
               labelVariant="h3"
             />
+            {/* <NavButton
+              route={contributeRoute}
+              label={VOLUNTEER}
+              labelVariant="h3"
+            /> */}
```

Comment:
Is this meant to be removed?

Reply:
It's supposed to be uncommented when this is merged with develop (I added the `contributeRoute` (/contribute) back in the release notes PR)